components/helper/ShareButtons.tsx

```diff
@@ -9,9 +12,131 @@ import {
   TwitterShareButton,
 } from 'react-share';
 
-export default function ShareButtons() {
+type ShareButtonsProps = {
+  popup: boolean;
+  setCopied: React.Dispatch<React.SetStateAction<boolean>>;
+  copied: boolean;
+};
+
+export default function ShareButtons({
+  popup, setCopied, copied,
+}: ShareButtonsProps) {
   const url = 'https://clearviction.org/calculator/head-initial-1-cont';
 
+  const style = {
+    fontSize: '1.25rem',
+    fontWeight: 400,
+    lineHeight: '1.375rem',
+    letterSpacing: '.0156rem',
+    textAlign: 'left',
+    color: '#2F3554',
+    textDecoration: 'underline',
+    marginLeft: '.4375rem',
+  };
+
+  const handleCopyClick = async () => {
+    await navigator.clipboard.writeText(url);
+    setCopied(true);
+  };
+
+  function handleEmailClick() {
+    const body = encodeURIComponent(url);
+    const mailtoUrl = `mailto:?body=${body}`;
+    window.open(mailtoUrl);
+  }
+
+  if (copied) {
+    return (
+      <Box sx={{
+        width: '20.3125rem',
+        height: '5.9375rem',
+        padding: '1rem',
+        borderRadius: '.375rem',
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'center',
+      }}
+      >
+        <Typography
+          variant="h5"
+          sx={{
+            fontFamily: 'Sintony',
+            fontWeight: 600,
+            textAlign: 'center',
+            marginBottom: '0',
+            fontSize: '1.5rem',
+
+          }}
+        >
+          Link Copied!
+
+        </Typography>
+      </Box>
+    );
+  }
+
+  if (popup) {
+    return (
+      <Box sx={{
+        width: '28.125rem', height: '20.625rem', padding: '1rem', borderRadius: '.375rem',
+      }}
+      >
+
+        <Box sx={{
+          display: 'flex', flexDirection: 'column', marginLeft: '1rem', gap: '.0094rem',
+        }}
+        >
+
+          <Typography
+            variant="h5"
+            sx={{
+              fontSize: '1.75rem',
+              fontWeight: 700,
+              lineHeight: '1.4375rem',
+              letterSpacing: '.0094rem',
+              textAlign: 'left',
+              marginTop: '1.5625rem',
+            }}
+          >
+            Share the Calculator
+          </Typography>
+
+          <Box sx={{
```

Comment:
This section should use conditional logic to display the different share options, instead of duplicating lines 141-160. If sometime we need to change the icons, we want to have to do it in as few places as possible.